pkg/report/diffcoverage.go

```diff
@@ -120,14 +120,15 @@ func (diff *diffCoverage) percentCovered() *Statistics {
 
 		case gittool.ModifyMode:
 
-			coverageProfile := generateCoverageProfileWithModifyMode(p, change)
-			coverageProfiles = append(coverageProfiles, coverageProfile)
-
-			node := diff.coverageTree.FindOrCreate(change.FileName)
-			node.TotalLines += int64(coverageProfile.TotalLines)
-			node.TotalCoveredLines += int64(coverageProfile.CoveredLines)
-			node.TotalViolationLines += int64(len(coverageProfile.TotalViolationLines))
-			node.CoverageProfile = coverageProfile
+			if coverageProfile := generateCoverageProfileWithModifyMode(p, change); coverageProfile != nil {
```

Comment:
unit test?

Reply:
it's already covered.

Reply:
then why the issue you found not detected?

Reply:
Add this because of best practice. For the input, it should not generate a nil pointer.